consul/acl_test.go

```diff
@@ -861,6 +862,73 @@ func TestACL_filterNodeDump(t *testing.T) {
 	}
 }
 
+func TestACL_filterPreparedQueries(t *testing.T) {
```

Comment:
This one should be covered - there's a check to make sure it marks the token as `<hidden>` when a management token isn't supplied.